.ci/on-schedule.sh

```diff
@@ -4,12 +4,17 @@ set -x
 
 # This script is triggered by a scheduled pipeline
 
+# Allow makepkg to work in a non-root environment & unbreak git
+chown -R nobody:root "$CI_BUILDS_DIR"
```

Comment:
I'm really not happy with using makepkg in this at all. This breaks GitHub actions completely.